copy/copy.go

```diff
@@ -109,7 +114,8 @@ func Copy(ctx context.Context, srcRoot, src, dstRoot, dst string, opts ...Opt) e
 		if err != nil {
 			return err
 		}
-		if err := c.copy(ctx, srcFollowed, dst, false); err != nil {
+		includeAll := len(c.includePatterns) == 0
```

Comment:
Excludepatterns don't matter in here?

Reply:
`includeAll` means we can skip evaluating the include patterns. Usually it's used when a parent dir matches an include pattern. It doesn't skip evaluating exclude patterns.

Maybe `skipIncludePatterns` would be a better name?